frontend/src/app/shared/form/base-form.component.spec.ts

```diff
@@ -0,0 +1,102 @@
+import { TestBed, ComponentFixture } from '@angular/core/testing';
+import { FormGroup } from '@angular/forms';
+import { BaseFormComponent } from './base-form.component';
+import { CaseFormatter } from '../format/case-formatter';
+import { TranslateModule } from '@ngx-translate/core';
+
+
+describe('BaseFormComponent', () => {
```

Comment:
rethink testing concept on this one
is it really the best way to pass an empty form for every test instead of creating a form and testing the reaction of the component based on the form state?